tests/sentry/api/endpoints/test_organization_dynamic_sampling_sdk_versions.py

```diff
@@ -211,27 +210,75 @@ def test_user_permissions_for_project_ids_filter(self):
         user = self.create_user("foo@example.com")
         self.login_as(user)
         with Feature({"organizations:server-side-sampling": True}):
-            response = self.client.get(f"{self.endpoint}?project={self.project.id}")
+            response = self.client.get(
+                f"{self.endpoint}?project={self.project.id}&"
+                f"start=2022-08-06T00:02:00+00:00&"
+                f"end=2022-08-07T00:00:02+00:00"
+            )
             assert response.status_code == 403
 
     def test_feature_flag_disabled(self):
         self.login_as(self.user)
-        response = self.client.get(self.endpoint)
+        response = self.client.get(
+            f"{self.endpoint}?project={self.project.id}&"
+            f"start=2022-08-06T00:02:00+00:00&"
+            f"end=2022-08-07T00:00:02+00:00"
+        )
         assert response.status_code == 404
 
     def test_no_project_ids_filter_requested(self):
         self.login_as(self.user)
         with Feature({"organizations:server-side-sampling": True}):
-            response = self.client.get(self.endpoint)
+            response = self.client.get(
+                f"{self.endpoint}?"
+                f"start=2022-08-06T00:02:00+00:00&"
+                f"end=2022-08-07T00:00:02+00:00"
+            )
             assert response.status_code == 200
             assert response.data == []
 
+    def test_no_query_start_or_no_query_end(self):
+        self.login_as(self.user)
+        response = self.client.get(
+            f"{self.endpoint}?project={self.project.id}&end=2022-08-07T00:00:02+00:00"
+        )
+        assert response.status_code == 400
+        assert response.json()["detail"] == "start and end are required"
+
+        response = self.client.get(
+            f"{self.endpoint}?project={self.project.id}&start=2022-08-06T00:02:00+00:00"
+        )
+        assert response.status_code == 400
+        assert response.json()["detail"] == "start and end are required"
+
+    def test_query_start_is_before_query_end(self):
+        self.login_as(self.user)
+        response = self.client.get(
```

Comment:
`with Feature({"organizations:server-side-sampling": True}):`

Reply:
Yes good catch .. completely forgot about it